src/ui/libs/DatalensChartkit/components/ChartKitBase/components/Header/components/ChartsInsights/components/TooltipContent/TooltipContent.tsx

```diff
@@ -35,10 +36,7 @@ export const TooltipContent = ({items, messages, onButtonClick}: TooltipContentP
                             {item.title}
                         </div>
                         {messages[item.locator] && (
-                            <div
-                                className={b('tooltip-message')}
```

Comment:
When you remove class with no other usages of it, you need to clean it from styles